static/docs/commands-reference/pull.md

```diff
@@ -1,53 +1,123 @@
 # pull
 
-Pulls data files to the working space.
+Pulls data files to the local workspace from a remote workspace.
 
-The set of data files to pull (usually it means downloading from the remote
-storage if file is not in the local cache yet) is determined by analyzing all
-`.dvc` files in the current branch, unless `--all-branches` is specified.
+## Usage
 
-The command pulls only output of a specific stage if dvc file is specified `dvc pull data.zip.dvc`.
+```usage
+    usage: dvc pull [-h] [-q | -v] [-j JOBS]
+                    [--show-checksums] [-r REMOTE]
+                    [-a] [-T] [-d] [-f] [-R]
+                    [targets [targets ...]]
+
+    Pull data files from the cloud.
+
+    positional arguments:
+      targets               DVC files.
+```
+
+## Description
+
+DVC supports sharing a data cache between workspaces.  Members of a team may
+be working with the same data, but using different algorithms, and will find
+it useful to share data.  The `dvc pull` command is analogous to `git pull`,
+and allows one to retrieve data from a remote cache.
+
+Specifically, the `dvc pull` command downloads files from a remote cache based
+on which have been updated in, or are new to, the remote cache.  By definition
+the `dvc pull` command will always interact between a local and remote cache.
```

Comment:
It's confusing a bit. Above we are saying that it pulls from a remote cache to the local workspace. Now a local cache is involved. I understand that both things are correct, but can we just write it a less confusing way?

Reply:
As I worked with rewriting this paragraph, it just seemed unnecessary so I've removed it entirely